cmd/botkube/main.go

```diff
@@ -17,8 +18,15 @@ func main() {
 	}
 	log.Logger.Info(fmt.Sprintf("Configuration:: %+v\n", Config))
 
-	sb := slack.NewSlackBot()
-	go sb.Start()
+	if Config.Communications.Slack.Enabled {
+		log.Logger.Info("Starting slack bot")
+		sb := slack.NewSlackBot()
+		go sb.Start()
 
-	controller.RegisterInformers(Config)
+		controller.RegisterInformers(Config)
+	} else if Config.Communications.Mattermost.Enabled {
+		log.Logger.Info("Starting mattermost bot")
+		mb := mattermost.NewMattermostBot()
+		mb.Start()
```

Comment:
This is not a go routine ?

Reply:
Yes it will be. Will update it in the final commit.

Reply:
No go routine yet ?

Reply:
We need to perform mandatory checks for connectivity/user/team/channel in Mattermost. So go routine will start in pkg/mattermost/mattermost.go